### PR TITLE
Update tutorial-shared-hosting.md

### PR DESCRIPTION
**Pull request by asmythe**
I'm guessing whoever wrote this was not a native English speaker. I cleaned it up and hopefully made it a little easier to follow.

### CODE REVIEW

File: docs/guide/tutorial-shared-hosting.md [hunk 1]
```diff
@@ -1,19 +1,34 @@
 Shared Hosting Environment
 ==========================
 
-Shared hosting environments are often quite limited about configuration and directory structure. Still in most cases
-you can run Yii 2.0 on these.
+Shared hosting environments are often quite limited about configuration and directory structure. Still in most cases you can run Yii 2.0 on a shared hosting environment with a few adjustements.
 
-Deploying basic application
+Deploying a basic application
```
Comment (samdark) [inline docs/guide/tutorial-shared-hosting.md]: It's not just a basic application. "Basic" here is application template name.

**Comment by samdark** [comment]
Thanks!

**Comment by asmythe** [comment]
I'm working on my first (and second) yii2 apps. I've been following these directions and can't get them to work even locally. I think my problem is here:

    www will be our frontend directory so move contents of frontend/web into it. Do the same with backend/web moving its contents into www/admin. **In each case you need to adjust paths in index.php and index-test.php.**

I don't know how I am supposed to adjust these paths. Any pointers would be much appreciated.

**Comment by samdark** [comment]
Well, these have to point to the file specified relatively to current dir.

**Comment by asmythe** [comment]
if my old structure was yii-application/frontend/web and my new structure is yii-application/public_html/web do I need to change anything?

I realized in the config file I need to point the controllerNamespace to public_html rather than front_end, but I do see the changes I need to make to index.php.

But if I go to http://localhost/yii-application/public_html/web/ <http://localhost/yii-application/public_html/web/> I get

exception 'yii\base\InvalidRouteException' with message 'Unable to resolve the request "site/error".' in /Users/asmythe/Dropbox/TF~Clients/Sites/RR/vendor/yiisoft/yii2/base/Module.php:461
Stack trace:

**Comment by samdark** [comment]
Yes, alias path is now different so you need to adjust in in config.

**Pull request merged**